include/oneapi/tbb/detail/_flow_graph_body_impl.h

```diff
@@ -396,6 +396,12 @@ class threshold_regulator<T, continue_msg, void> : public continue_receiver, no_
         return my_node->decrement_counter( 1 );
     }
 
+#if __TBB_PREVIEW_FLOW_GRAPH_TRY_PUT_AND_WAIT
+    graph_task* execute(const message_metainfo&) override {
+        return execute();
```

Comment:
Please add a comment here why the metainfo is dropped.

Reply:
Fixed